### PR TITLE
Convert spaces to tabs

### PR DESCRIPTION
**Pull request by Bituvo**
This PR refactors some scripts in `/src`, in that it replaces space indentation with tab indentation. Additionally, the 2 tab continuation rule is abided by. For a lot of these files, this was the first time they have been touched in years.

Like my "Remove trailing whitespace" PR, feel free to review. Or not.

### CODE REVIEW

**Comment by appgurueu** [comment]
This is something that should ideally be caught by CI as well.

File: src/network/networkprotocol.h [hunk 1]
```diff
@@ -233,8 +233,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // Constant that differentiates the protocol from random data and other protocols
 #define PROTOCOL_ID 0x4f457403
 
-#define PASSWORD_SIZE 28       // Maximum password length. Allows for
-                               // base64-encoded SHA-1 (27+\0).
+#define PASSWORD_SIZE 28	// Maximum password length. Allows for
+							// base64-encoded SHA-1 (27+\0).
```
Comment (Desour) [inline src/network/networkprotocol.h:237]: Please don't use tabs after text. The spaces here are alignment, not indentation.

File: src/pathfinder.cpp [hunk 2]
```diff
@@ -128,12 +128,12 @@ class PathGridnode {
...
-	                                        * s = surface (walkable node)
-	                                        * - = non-walkable node (e.g. air) above surface
-	                                        * g = other non-walkable node
-                                                */
+											* u = unknown
+											* i = invalid
+											* s = surface (walkable node)
+											* - = non-walkable node (e.g. air) above surface
+											* g = other non-walkable node
+											*/
```
Comment (Desour) [inline src/pathfinder.cpp:136]: Also here, it's alignment, not indentation.

File: src/script/common/c_content.h
Comment (Desour) [inline src/script/common/c_content.h:1]: Same in this file.

File: src/script/common/c_converter.h
Comment (Desour) [inline src/script/common/c_converter.h:1]: And here.

**Review by Desour — COMMENTED** [review]
Fine otherwise.
CI can come afterwards.

File: src/script/common/c_converter.h
Comment (Bituvo) [reply src/script/common/c_converter.h:1]: Can you show me where?

Comment (Desour) [reply src/script/common/c_converter.h:1]: You've removed the alignment of the function declarations. Same in `c_content.h`.

(It's not like I the alignment that much there, but it's outside of this PR's scope. If you insist on doing it, please put that changes into a separate commit.)

Comment (Bituvo) [reply src/script/common/c_converter.h:1]: Will fix.

File: src/script/common/c_content.h
Comment (Bituvo) [reply src/script/common/c_content.h:1]: Will fix.

File: src/script/common/c_content.h [hunk 3]
```diff
@@ -71,137 +71,104 @@ struct collisionMoveResult;
...
-void               read_server_sound_params  (lua_State *L, int index,
-                                              ServerPlayingSound &params);
+void    read_server_sound_params (lua_State *L, int index, ServerPlayingSound &params);
 
-void               push_dig_params           (lua_State *L,
-                                              const DigParams &params);
-void               push_hit_params           (lua_State *L,
-                                              const HitParams &params);
+void    push_dig_params          (lua_State *L, const DigParams &params);
+void    push_hit_params          (lua_State *L, const HitParams &params);
```
Comment (sfan5) [inline src/script/common/c_content.h:90]: are tabs being used for aligment here or what are these changes?

**Review by Desour — APPROVED** [review]
LGTM.

(The alignment changes in `c_content.h` and `c_converter.h` are a bit out of scope, but I don't really care. File-wide alignment isn't common in our code style anyways.)

File: src/script/common/c_content.h (hunk 3 above)
Comment (SmallJoker) [reply src/script/common/c_content.h:90]: There were no tabs in the previous code. The alignment was okay and does IMO not need any fixing.

Comment (Zughy) [reply src/script/common/c_content.h:90]: @Bituvo please take care of the review so we can merge it

Comment (Bituvo) [reply src/script/common/c_content.h:90]: I'll revert the alignment changes I made. I may or may not move those to a separate PR

**Comment by Desour** [comment]
Approval holds.

**Review by nerzhul — APPROVED** [review]
lgtm

**Pull request merged**